trackintel/model/trips.py

```diff
@@ -65,6 +67,15 @@ def _validate(obj):
             obj["finished_at"]
         ), "dtype of finished_at is {} but has to be datetime64 and timezone aware".format(obj["finished_at"].dtype)
 
+        # Check geometry if Trips is a GeoDataFrame
+        if "geom" in obj.columns:
```

Comment:
At the moment, checking `obj.geometyr is None` only leads to an error `AttributeError: No geometry data set yet`. Furthermore, it seems that geopandas will at some point be updated to return a dataframe if no geometry column exists (https://github.com/geopandas/geopandas/issues/1911). This is problematic for us, since the code now is designed in a way that the geometry column is optional, but trips is still always a GeoDataFrame. Since this is already tricky, and will probably become impossible with new geopandas versions, I'd maybe suggest to go back to the option we previously considered, where trips can be either a GeoDataframe or a DataFrame? Let me know what you think.

Reply:
Ah, I see. Yes in this case it is better to go for your initial idea. In the case it is a geodataframe we still need to address the geometry column dynamically.